### PR TITLE
TEST ADDING NEW COUNTERS WITH DIFFERENT VALUES AND THEN DELETE ALL COUNTERS

### PR DESCRIPTION
**Pull request by Dmitrii126776**

### CODE REVIEW

File: test/spec/kuzhilin/testAddDeleteCounters.spec.js [hunk 1]
```diff
@@ -1,89 +1,110 @@
...
-  it('should  add new Third counter with value 50', function () {
-    let newName = 'Third counter';
-    browser.$('//input[@name="name"]').setValue(newName);
-    browser.$('//input[@name="value"]').setValue('50');
-    browser.$('//button[text()="Add Counter"]').click();
-    let result = browser.$('//h3[text()[1]="3. " and text()[2]="'+newName+'"]').isDisplayed();
-    expect(result).eq(true);
+  it('should input by click 1. Default Counter (8, -5)', function () {
+    browser.$('(//button[@step="8"])[1]').click();
+    browser.$('(//button[@step="-5"])[1]').click();
```
Comment (ludatkach) [inline test/spec/kuzhilin/testAddDeleteCounters.spec.js:26]: add waitUntil to convert it to test

File: test/spec/kuzhilin/testAddDeleteCounters.spec.js [hunk 2]
```diff
@@ -1,89 +1,110 @@
...
     browser.$('(//button[@name="negative"])[2]').click();
     browser.waitUntil( () => browser.$('(//input[@name="lower"])[2]').isDisplayed());
     browser.$('(//input[@name="upper"])[2]').setValue('9');
   });
 
-  it('should click ULF and LLF into 3. THIRD COUNTER get value by default and then input ULF 9', function () {
+  it('should input by click 2. Second Counter (-7, 3)', function () {
+    browser.$('(//button[@step="-7"])[2]').click();
+    browser.$('(//button[@step="3"])[2]').click();
+  });
```
Comment (ludatkach) [inline test/spec/kuzhilin/testAddDeleteCounters.spec.js:57]: add waitUntil here

File: test/spec/kuzhilin/testAddDeleteCounters.spec.js [hunk 3]
```diff
@@ -1,89 +1,110 @@
...
     browser.waitUntil( () => browser.$('(//input[@name="upper"])[3]').isDisplayed());
     browser.$('(//button[@name="negative"])[3]').click();
     browser.waitUntil( () => browser.$('(//input[@name="lower"])[3]').isDisplayed());
     browser.$('(//input[@name="upper"])[3]').setValue('9');
   });
 
+  it('should input by click 3. Third Counter (2, -6)', function () {
+    browser.$('(//button[@step="2"])[3]').click();
+    browser.$('(//button[@step="-6"])[3]').click();
+  });
```
Comment (ludatkach) [inline test/spec/kuzhilin/testAddDeleteCounters.spec.js:90]: add waitUntil here

File: test/spec/kuzhilin/testAddDeleteCounters.spec.js (hunk 1 above)
Comment (Dmitrii126776) [reply test/spec/kuzhilin/testAddDeleteCounters.spec.js:26]: I want to discuss about this at our meeting today

Comment (ludatkach) [reply test/spec/kuzhilin/testAddDeleteCounters.spec.js:26]: Sure  we will . You can merge now

**Pull request merged**